index.html

```diff
@@ -236,13 +238,25 @@ <h4 class="heading-langS">Skills</h4>
                                                   <img src="./resources/getintouch.svg" alt="getintouch">
                                                 </button>
                                               </form>
+                                              <div class="bottom-right">
+                                                <img src="./resources/contact form-illustratoin-3-desktop.svg" alt="bottom-right">
+                                              </div>
                                             </div>
                                              <!-- contact form End-->
+                                            <div class="desktop-img-bottom">
+                                            <div class="bottomleft">
+                                              <img src="./resources/bottomleft.svg" alt="bottomleft">
+                                              </div>
+                                              <div class="bottom1">
+                                                <img src="./resources/bottom1.svg" alt="bottom1">
+                                              </div>
+                                              </div>
```

Comment:
- [ ] Kindly fix the issue of this background as suggested by the previous reviewer.

Reply:
[Optional] kindly increase the width of the horizontal line close to the footer social icons